package.json

```diff
@@ -78,7 +78,7 @@
   "scripts": {
     "lint": "eslint --ignore-path .gitignore . --ext ts --ext tsx --ext js --ext jsx",
     "lint:fix": "yarn lint --fix",
-    "build": "gatsby build",
+    "build": "gatsby build --prefix-paths",
```

Comment:
Please create a `build:staging` with this flag so we can keep the regular `build` without the `--prefix-paths` option for the production build